src/source/reference/post-metrics-data.rst

```diff
@@ -4,6 +4,23 @@
 POST /metrics/data
 =======================
 
+V2
+~~~
+.. list-table::
+ :widths: 20 80
+
+ * - Scope
+   - ``console_ro`` or ``api_ro``
+
+ * - Accept
+   - ``application/json``
+ * - Content-Type
+   - ``application/sellside.metrics.data-v2+json``
+
+Version 2 is replacing CPC values with BidMicros and BilledMicros. One cent equals to 10000 micros. 
```

Comment:
I think to the API partners completely unfamiliar with what is the purpose of these two things suddenly popping up as a replacement of CPC, it will be still confusing. Why two values, when are they different? why introduce micros in the first place?

Reply:
also, it's not only about CPC, is it? how about spent?

Reply:
can we expand on that? `for future purposes` is not really a big incentive for anytone to start migrating to the next version. We will get the question of why differentiation between bid and billed for sure, we might as well pre-answer it in a short paragraph, than risk that the next sheriff will confuse them even more or ping the seller team for an answer again and again. Something around adjusting value, and autobiding...